### ISSUE TITLE
Focusing previously focused notification does not announce their content

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#188762]
Testing #188528

Repro:

1. Enable screen reader mode
2. Click screen reader optimized status bar button to trigger a notification
3. Run the Show Notifications command, content is announced
4. Focus the editor
5. Run the Show Notifications command, content is _not_ announced

![Recording 2023-07-25 at 03 57 32](https://github.com/microsoft/vscode/assets/2193314/e24417b9-a41e-4896-a5c0-7e44fd7bce16)

**Comment by Tyriar** [comment]
It's also a little strange that it's announced when I focus Yes, but not when I press shift+tab while Yes is focused

**Comment by meganrogge** [comment]
I don't think this is a new issue

**Comment by bpasero** [comment]
@meganrogge there is a subtle difference between running "Show Notifications" when the notifications are not visible vs. when they are visible.

When they are NOT visible, we not only open notifications but we also focus the first entry. This makes the screen reader announce it.

When they are visible, we only focus the notifications container, but not the first notification. Arguably that is a bit inconsistent. However, for a screen reader you can easily get to the first notification when you arrow down.

I am fine changing this to also focus the first notification when the center is visible and you run the command if you think that makes it more accessible?

**Comment by meganrogge** [comment]
@bpasero thanks for that insight. I don't think there's any downside to focusing the first notification automatically, since a user will have to do that manually themselves even if they're only interested in the second or third one.
